cypress/integration/show_work_orders.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+beforeEach(() => {
+  cy.fixture('work_orders/jobs.json').as('workorderslist')
+  cy.server()
+  cy.visit('/')
+})
+
+describe('Show all work orders', () => {
+  it('Displays work order details, reference, date raised, last update, priority, property and description', () => {
+    //Stub request with work orders response
+
+    cy.route('GET', 'api/work-orders', '@workorderslist')
+    cy.visit('/work-orders/jobs')
+
+    cy.get('.govuk-table').within(() => {
+      cy.contains('th', 'Reference')
+      cy.contains('th', 'Date raised')
+      cy.contains('th', 'Last update')
+      cy.contains('th', 'Priority')
+      cy.contains('th', 'Property')
+      cy.contains('th', 'Description')
+    })
+
+    //can see reference number
+    cy.get('.govuk-table__cell').within(() => {
+      cy.contains('00012345')
+    })
+
+    //can see date raised
+    cy.get('.govuk-table__cell').within(() => {
+      cy.contains('13 January 2021')
+    })
+
+    //can see last updated
+    cy.get('.govuk-table__cell').within(() => {
+      cy.contains('13 January 2021')
```

Comment:
I think this should be different to the date raised (above). 

Ideally we'd scope all of our searches to within a particular column of the table. I'd need to look up how to do that in Cypress.